packages/world/src/systems/libraries/MoveLib.sol

```diff
@@ -61,10 +62,14 @@ library MoveLib {
       if (gravityAppliesForMove) {
         if (oldBaseCoord.y() < newBaseCoord.y()) {
           numJumps++;
+          currentFallHeight = 0;
```

Comment:
I think we might not need this new variable, can't we just track the total falls and then take the min(numFalls - threshold, 0)?

Reply:
but we want consecutive falls, if we just track `numFalls` like before, we won't know if they fell one by one which would result in no damage, or 4 in one go

Reply:
ah ok